src/h3lib/lib/polyfill.c

```diff
@@ -692,9 +696,27 @@ void iterDestroyPolygon(IterCellsPolygon *iter) {
 H3Error H3_EXPORT(polygonToCellsExperimental)(const GeoPolygon *polygon,
                                               int res, uint32_t flags,
                                               H3Index *out) {
+#ifdef H3_POLYGON_TO_CELLS_ASSERT
```

Comment:
Should we comment on how to set this variable?  Can we use  the `TESTONLY` macro here instead?